docs/privacy.md

```
@@ -75,19 +72,20 @@ Remove-Item "$HOME\.codex\multi-auth\logs\codex-plugin" -Recurse -Force -ErrorAc
 - https://openai.com/policies/terms-of-use/
 - https://openai.com/policies/privacy-policy/
 
-* * *
+---
 
 ## Related
 
 - [configuration.md](configuration.md)
 - [troubleshooting.md](troubleshooting.md)
 - [reference/storage-paths.md](reference/storage-paths.md)
+- [../SECURITY.md](../SECURITY.md)
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)